MM2RandoLib/Randomizers/RText.cs

```diff
@@ -444,308 +436,23 @@ static char GetBossWeaknessDamageChar(int dmg)
             0x001A10, // "C"
         };
 
-        public static Dictionary<char, byte> CreditsCipher = new Dictionary<char, byte>()
-        {
-            { ' ', 0x00},
-            { 'A', 0x01},
-            { 'B', 0x02},
-            { 'C', 0x03},
-            { 'D', 0x04},
-            { 'E', 0x05}, 
-            { 'F', 0x06},
-            { 'G', 0x07},
-            { 'H', 0x08},
-            { 'I', 0x09}, 
-            { 'J', 0x0A}, 
-            { 'K', 0x0B}, 
-            { 'L', 0x0C}, 
-            { 'M', 0x0D}, 
-            { 'N', 0x0E},
-            { 'O', 0x0F},
-            { 'P', 0x10},
-            { 'Q', 0x11},
-            { 'R', 0x12},
-            { 'S', 0x13},
-            { 'T', 0x14},
-            { 'U', 0x15},
-            { 'V', 0x16},
-            { 'W', 0x17},
-            { 'X', 0x18},
-            { 'Y', 0x19},
-            { 'Z', 0x1A},
-            { '.', 0x1C},
-            { ',', 0x1D},
-            {'\'', 0x1E},
-            { '!', 0x1F},
-            { 'f', 0x20},
-            { '0', 0x30},
-            { '1', 0x31},
-            { '2', 0x32},
-            { '3', 0x33},
-            { '4', 0x34},
-            { '5', 0x35},
-            { '6', 0x36},
-            { '7', 0x37},
-            { '8', 0x38}, 
-            { '9', 0x39},
-            { '=', 0x23},
-        };
-
-        public static Dictionary<char, byte> IntroCipher = new Dictionary<char, byte>()
-        {
-            { ' ', 0x00 },
-            { '0', 0xA0 },
-            { '8', 0xA1 },
-            { '2', 0xA2 },
-            { '©', 0xA3 },
-            { 't', 0xA4 }, // tm
-            { '9', 0xA5 },
-            { '7', 0xA6 },
-            { '1', 0xA7 },
-            { '3', 0xA8 }, // check
-            { '4', 0xA9 }, // check
-            { '5', 0xAA }, // check
-            { '6', 0xAB }, // check
-            { '|', 0xC0 }, // Blank space, change later? also, there may be a blank row (B) to use...
-            { 'A', 0xC1 },
-            { 'B', 0xC2 },
-            { 'C', 0xC3 },
-            { 'D', 0xC4 },
-            { 'E', 0xC5 },
-            { 'F', 0xC6 },
-            { 'G', 0xC7 },
-            { 'H', 0xC8 },
-            { 'I', 0xC9 },
-            { 'J', 0xCA },
-            { 'K', 0xCB },
-            { 'L', 0xCC },
-            { 'M', 0xCD },
-            { 'N', 0xCE },
-            { 'O', 0xCF },
-            { 'P', 0xD0 },
-            { 'Q', 0xD1 },
-            { 'R', 0xD2 },
-            { 'S', 0xD3 },
-            { 'T', 0xD4 },
-            { 'U', 0xD5 },
-            { 'V', 0xD6 },
-            { 'W', 0xD7 },
-            { 'X', 0xD8 },
-            { 'Y', 0xD9 },
-            { 'Z', 0xDA },
-            { '?', 0xDB }, // "r." change to ?
-            { '.', 0xDC },
-            { ',', 0xDD },
-            { '\'', 0xDE },
-            { '!', 0xDF },
-        };
         // STAFF == D3 D4 C1 C6 C6
```

Comment:
Is this comment dangling now? Should it have been moved too?

Reply:
I left that comment for now, because I am not sure what it meant... I think it has something to do with the x-position of the text in the credits roll.

Reply:
I think it's the encoding of the letters, notice both are 5 in length and the F F = C6 C6

Reply:
yeah, I figured that too. Let me remove the comment, or at least put it somewhere that makes sense.

Reply:
Done.